packages/expo/cli/utils/__tests__/downloadExpoGoAsync-test.ts

```diff
@@ -0,0 +1,138 @@
+import { vol } from 'memfs';
+import nock from 'nock';
+
+import { getExpoApiBaseUrl } from '../../api/endpoint';
+import { downloadAppAsync } from '../downloadAppAsync';
+import { downloadExpoGoAsync } from '../downloadExpoGoAsync';
+import { extractAsync } from '../tar';
+
+const asMock = (fn: any): jest.Mock => fn;
+
+jest.mock('fs');
+
+jest.mock('progress', () => ({
```

Comment:
May want to create `expo/cli/__mocks__/progress.ts` and mock it there if we expect tests to want these modules mocked by default